### PR TITLE
added the changes 

### PR DESCRIPTION
**Pull request by daksh1210jain**

### CODE REVIEW

File: dcodeKi-app/dcodeKi.kv [hunk 1]
```diff
@@ -3,6 +3,21 @@
...
+                pos: self.pos
+                size: self.size
+                source: 'wave.png'
+	BoxLayout:
+        orientation: 'vertical'
+        Label:
+            text: 'Symbol Cipher Page'
+        Button:
+            text: 'Back to Main'
+            on_press: app.screen_manager.current = 'main'
```
Comment (Pankaj0038) [inline dcodeKi-app/dcodeKi.kv]: I think you want to call the show_main_screen function , 
```
on_press: app.show_main_screen(5) # delay time 5s 
```

background image doesn't exist , if you want to add any image then add that also in your pr

File: dcodeKi-app/dcodeKi.py [hunk 2]
```diff
@@ -141,15 +144,16 @@ def copy(self):
 		except:
 			self.ids.string.text = "Nothing to copy!"
 
-
+class SymbolCipherPage(Screen):
```
Comment (Pankaj0038) [inline dcodeKi-app/dcodeKi.py]: name of the class in python file and kivy file doesn't match 
make it "SymbolCipherScreen"

while adding the widget use the same

**Comment by daksh1210jain** [comment]
sir made the changes now kindly have a look and merge my PR

**Comment by Pankaj0038** [comment]
well done @daksh1210jain

**Pull request merged**